presto-main/src/main/java/com/facebook/presto/operator/aggregation/PrecisionRecallAggregation.java

```diff
@@ -181,46 +182,46 @@ public BucketResult(
             return Collections.<BucketResult>emptyList().iterator();
         }
 
-        final double totalTrueWeight = Streams.stream(state.getTrueWeights().iterator())
-                .mapToDouble(c -> c.weight)
+        double totalTrueWeight = Streams.stream(state.getTrueWeights().iterator())
+                .mapToDouble(FixedDoubleHistogram.Bucket::getWeight)
```

Comment:
Usually these refactors goes into a separate commit.  But if it's too much burden to do it now we can skip it this time.